recipes/ui.rb

```diff
@@ -72,9 +72,12 @@
       false
     end
 
-  common_name = node['cdap']['security']['ssl_common_name']
-  keypath = node['cdap']['cdap_site']['dashboard.ssl.key']
-  certpath = node['cdap']['cdap_site']['dashboard.ssl.cert']
+  if ssl_enabled.to_s == 'true'
```

Comment:
this should be redundant to the guard, but the guard (not shown in the diff) has a bug in it.  it's missing the ``.to_s == 'true'``